src/commands/sqlite.rs

```diff
@@ -232,15 +271,42 @@ impl ListCommand {
                 return Ok(());
             }
         }
-
         match self.group_by {
-            GroupBy::App => print_apps(links, databases_without_links),
-            GroupBy::Database => print_databases(links, databases_without_links),
+            Some(GroupBy::App) => print_apps(links, databases_without_links),
+            Some(GroupBy::Database) => print_databases(links, databases_without_links),
+            None => print_apps(links, databases_without_links),
         }
         Ok(())
     }
 }
 
+fn json_list_format(database: Database) -> DatabasesListJson {
+    DatabasesListJson {
+        database: database.name,
+        links: database
+            .links
+            .into_iter()
+            .map(|l| SimplifiedResourceLabel {
+                label: l.label,
+                app: l.app_name.unwrap_or("UNKNOWN".to_string()),
+            })
+            .collect(),
+    }
+}
+
+#[derive(Serialize)]
+struct DatabasesListJson {
+    database: String,
+    links: Vec<SimplifiedResourceLabel>,
+}
```

Comment:
Nit: there shouldn't be a need to have owned strings here:
```suggestion
#[derive(Serialize)]
struct DatabasesListJson<'a> {
    database: &'a str,
    links: Vec<SimplifiedResourceLabel<'a>>,
}
```

Such a change would require you also change this:

```rust
fn json_list_format<'a>(database: &'a Database) -> DatabasesListJson<'a> {
```